README.rst

```diff
@@ -14,9 +16,47 @@ The following GUI backends are supported:
 - PyQt
 - PySide
 
+**Warning:** In TraitsUI version 5.0 the default GUI backend will change from
```

Comment:
fair enough (especially when it does not look that it will arrive soon https://github.com/github/markup/issues/68)